app/controllers/v0/letters_generator_controller.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'lighthouse/letters_generator/service'
+require 'lighthouse/letters_generator/service_error'
+
+module V0
+  class LettersGeneratorController < ApplicationController
```

Comment:
In the EVSS version of this controller, I noticed this line:

```ruby
before_action { authorize :evss, :access_letters? }
```

Is this needed? What does this line do?

Reply:
That line checks that the current user has the correct IDs needed to make calls to EVSS before calling any methods on the controller. I think one of the main uses is to prevent avoidable errors associated with missing IDs, but it also acts in some ways as an authentication layer, because if a user is not logged in, they won't have the necessary IDs either

Reply:
Cool - since we're using Lighthouse, and Lighthouse only requires an ICN, would you say this link is not needed here @jerekshoe?

Reply:
It depends on if you want this controller to be authenticated or not. I'm planning on adding it to the benefits claims controller soon, but you guys don't have to add it to the controller here

Reply:
I think we probably would. I sorta misread your earlier comment. @samcoforma?